tpls/desul/include/desul/atomics/Lock_Array_Cuda.hpp

```diff
@@ -138,42 +138,33 @@ namespace {
 static int lock_array_copied = 0;
 inline int eliminate_warning_for_lock_array() { return lock_array_copied; }
 }  // namespace
-
-#ifdef __CUDACC_RDC__
-inline
-#else
-static
```

Comment:
can we try instead of reverting here: a namespace { and then close that namespace again? It looks like something is lost here anything.

Reply:
tried this myself too I think we should revert